chart-bar.html

```diff
@@ -48,7 +48,9 @@
 
     <style include="chart-styles"></style>
 
-    <canvas id="canvas"></canvas>
+    <div>
+      <canvas id="canvas"></canvas>
+    </div>
```

Comment:
Weren't these changes merged in a previous pull request?
https://github.com/robdodson/chart-elements/blob/master/chart-bar.html#L51-L53